firebase-functions/functions/src/operations/GameOperations.ts

```diff
@@ -8,30 +8,38 @@ import { Result } from '../models/result';
 import { Round } from '../models/round';
 import { Game } from '../models/game';
 import { collections } from '../constants/collections';
-import { Room } from '../models/room';
-import { GameType } from '../models/game-type';
 import DocumentData = firestore.DocumentData;
 import QueryDocumentSnapshot = firestore.QueryDocumentSnapshot;
+import { getRoom } from './roomOperations';
 
-export async function addRoundToGame(roomId: string, players: QueryDocumentSnapshot<DocumentData>[]): Promise<void> {
+export async function getCurrentGame(roomId: string, players: QueryDocumentSnapshot<DocumentData>[]): Promise<Game> {
```

Comment:
I think this method name is really misleading. It doesn't merely get the game, it adds a round to the current game or creates a new one. 
I think the old name was better. I suggest you either:
- create a second method that returns the current game: this keeps the method signature honest and splits between methods that change state and methods that return something (CQRS)
- rename it back and just still return the game: I think that is the worse solution, but less convoluted

Reply:
About your:
1st suggestion: one of the main focuses in this refactoring is to reduce as possible firebase DB calls, spliting means different calls as we don't have a shared store.
2nd suggestion: `addRoundToGame`  and `getCurrentGame` as equally convoluted :), nothing makes it less convoluted :)

I have changed it tho to `addRoundToGame` so I won't hold this PR for long, until we find a more suitable name.

Reply:
Thanks, you are right with that first argument.

About the second: the problem with `getCurrentGame` is that it doesn't indicate that something _changes_. You call it and expect to get a game, but in reality you add a round to a game and even create a new one possibly. That is some thick side effect for a `get...` method. 

Anyway, that was just for completeness :) Changing the name back works for me, thanks.